tuf/cmd/upload_secrets.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/GoogleCloudPlatform/runtimes-common/ctc_lib"
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/config"
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/deployer"
+
+	"github.com/spf13/cobra"
+)
+
+// Flags
+var rootKey = "~/root.json"
```

Comment:
Could we make the home directory explicit here? Something like https://github.com/mitchellh/go-homedir can get the homedir, and we can join the paths.